services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/directory/decorator/indexing/IndexingDecoratorOnDisk.java

```diff
@@ -20,6 +18,7 @@ public IndexingDecoratorOnDisk(IndexDirectory<T, WritableOnDisk> directory, Stri
     super(directory, indexFileName);
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
@pithumke  
`JSONArray.addAll(elements)` - is causing the warning, which was there already

just wanted to mitigate the warning in this case, because I don't see a way to fix it in other ways